docs/samples/bookinfo.md

```diff
@@ -170,97 +132,97 @@ route requests to all available versions of a service in a random fashion.
-   ---
-   kind: route-rule
-   name: reviews-default
-   namespace: default
-   spec:
-     destination: reviews.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   kind: route-rule
-   name: details-default
-   namespace: default
-   spec:
-     destination: details.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   kind: route-rule
-   name: productpage-default
-   namespace: default
-   spec:
-     destination: productpage.default.svc.cluster.local
-     precedence: 1
-     route:
-     - tags:
-         version: v1
-       weight: 100
-   ---
-   ```
-
-   > Note: In the current Kubernetes implemention of Istio, the rules are stored in ThirdPartyResources.
-   > You can look directly at the stored rules in Kubernetes using the `kubectl` command. For example,
-   > the following command will display all defined rules:
-   > ```bash
-   > $ kubectl get istioconfig -o yaml
-   > ```
+```bash
+istioctl create -f \
+    https://raw.githubusercontent.com/istio/istio/master/demos/apps/bookinfo/route-rule-all-v1.yaml
+```
+
+You can display the routes that are defined with the following command:
+
+```bash
+$ istioctl get route-rules -o yaml
+kind: route-rule
+name: ratings-default
+namespace: default
+spec:
+ destination: ratings.default.svc.cluster.local
+ precedence: 1
+ route:
+ - tags:
+     version: v1
+   weight: 100
+---
+kind: route-rule
+name: reviews-default
+namespace: default
+spec:
+ destination: reviews.default.svc.cluster.local
+ precedence: 1
+ route:
+ - tags:
+     version: v1
+   weight: 100
+---
+kind: route-rule
+name: details-default
+namespace: default
+spec:
+ destination: details.default.svc.cluster.local
+ precedence: 1
+ route:
+ - tags:
+     version: v1
+   weight: 100
+---
+kind: route-rule
+name: productpage-default
+namespace: default
+spec:
+ destination: productpage.default.svc.cluster.local
+ precedence: 1
+ route:
+ - tags:
+     version: v1
+   weight: 100
+---
+```
+
+> Note: In the current Kubernetes implemention of Istio, the rules are stored in ThirdPartyResources.
+> You can look directly at the stored rules in Kubernetes using the `kubectl` command. For example,
+> the following command will display all defined rules:
+> ```bash
+>kubectl get istioconfig -o yaml
+> ```
 
    Since rule propagation to the proxies is asynchronous, you should wait a few seconds for the rules
    to propagate to all pods before attempting to access the application.
 
-   If you open the Bookinfo URL (`http://$GATEWAY_URL/productpage`) in your browser,
-   you should see the bookinfo application `productpage` displayed. Notice that the `productpage`
-   is displayed with no rating stars since `reviews:v1` does not access the ratings service.
+   Notice that the `productpage` is displayed with no rating stars since `reviews:v1` does not access the ratings service.
```

Comment:
It is already present in the previous section, where it makes sense. The user has to first check  the bookinfo page is displayed, before configuring routing rules.